docs/source/tutorial.rst

```diff
@@ -145,17 +145,71 @@ They can be set in a way similar to the compiler_options::
 Setting up a series of simulations
 ----------------------------------
 
-The ``experiment_definition`` created could be modified to set up a series of simulations by specifying the
-the parameter to sweep and the range by calling the ``with_modifiers()`` method on the ``experiment_definition``::
+The ``experiment_definition`` created could be modified to set up a series of simulations following either of 
+approaches listed below:
```

Comment:
this sentence feels a bit weird, maybe "Following either of the approaches listed below, the created experiment definition can be modified to set up a series of simulations" or something